script.js

```diff
@@ -0,0 +1,43 @@
+
+
+
+$(function(event) {
+// function for when button is clicked
+console.log("dom");
+// global variables
+var cross = true;
+var winner = false;
+
+function buttonClick() {
+  $('td').click(function() {
+    check();
+    if(cross === true && winner === false) {
+      cross = false;
+      $(this).html('X').addClass('X');
+      $('.playerTurn').html('It is O\'s turn');
+      console.log(this);
+    } else if(winner === false){
+      cross = true;
+      $(this).html('O').addClass('O');
+      $('.playerTurn').html('It is X\'s turn');
+    }
+  });
+}
+
+// Checks for line
+function check() {
+  debugger;
+  if(($('td')[0].innerHTML === $('td')[1].innerHTML === $('td')[2].innerHTML) || ($('td')[3].innerHTML === $('td')[4].innerHTML === $('td')[5].innerHTML) || ($('td')[6].innerHTML === $('td')[7].innerHTML === $('td')[8].innerHTML) || ($('td')[0].innerHTML === $('td')[3].innerHTML === $('td')[6].innerHTML) || ($('td')[1].innerHTML === $('td')[4].innerHTML === $('td')[7].innerHTML) || ($('td')[2].innerHTML === $('td')[5].innerHTML === $('td')[8].innerHTML) || ($('td')[2].innerHTML === $('td')[4].innerHTML === $('td')[6].innerHTML) || ($('td')[0].innerHTML === $('td')[4].innerHTML === $('td')[8].innerHTML)) {
```

Comment:
If you want to check if multiple conditions are true e.g. if cells 0, 1 and 2 are all x, you need to use the && operator; For example:

if(($('td')[0].innerHTML === $('td')[1].innerHTML && $('td')[0].innerHTML) === $('td')[1].innerHTML))

And so on - you would need to check literally every possible combination against each other. Not a bad idea, but definitely easier ways to do this.